packages/terra-dev-site/src/modals/_ApplicataionSwitcherModal.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import List, { Item } from 'terra-list';
+import Hyperlink from 'terra-hyperlink';
+import ApplicationModal from '@cerner/terra-application/lib/application-modal/ApplicationModal';
```

Comment:
this file doesn't exist yet, but it will before we enable modals in terra dev site.